### PR TITLE
 bugfix/13574-packedbubble-size-with-multiple-series 

### LINKED ISSUE #13574: Packed bubble - additional series has an impact on bubble size
#### Expected behaviour

The bubble size should be constant, no matter of other additional series.

#### Actual behaviour

The bubble size is changing when other series has been added. 

#### Live demo with steps to reproduce

With Pie series: https://jsfiddle.net/BlackLabel/4pba0jns/
Without Pie series: https://jsfiddle.net/BlackLabel/n2pr5v6k/

Compare the size of bubbles in the demos to see the difference. It seems that more series = smaller bubbles.

#### Product version

Highcharts JS v8.1.0 (2020-05-05)

### PR DESCRIPTION
**Pull request by Izothep**
Fixed #13574, packedbubble size changed when other series were added.

### CODE REVIEW

**Comment by sebastianbochan** [comment]
Looks good, but the unit-test is missed.

**Comment by Izothep** [comment]
I wanted to add it in separate commit.

File: samples/unit-tests/series-packedbubble/update/demo.js [hunk 1]
```diff
@@ -22,9 +24,21 @@ QUnit.test('Series update', function (assert) {
...
     );
+    chart.addSeries({
+        type: 'pie',
+        data: [1],
+        size: '5%'
+    });
+    assert.strictEqual(
+        radius,
+        point.radius,
+        "point's radius is not changing after adding series other than packedbubble."
```
Comment (sebastianbochan) [inline samples/unit-tests/series-packedbubble/update/demo.js:42]: You need describe what should happen.

I.e "Point radius should be updated ..."

Comment (Izothep) [reply samples/unit-tests/series-packedbubble/update/demo.js]: Done.

**Pull request merged**